osm_to_sandbox/osm_to_sandbox.py

```diff
@@ -371,5 +395,41 @@ def main():
     print('Done.')
 
 
+def cli():
+    parser = argparse.ArgumentParser(
+        description="Downloads data from Overpass API and uploads it to the mapping "
+                    "sandbox.",
+        epilog="Because sandboxes are for grown-ups, too!",
+    )
+    parser = add_args(parser)
+    args = parser.parse_args()
+    args.bbox = [float(x.strip()) for x in args.bbox.split(',')]
+    main(args.bbox, args.auth_header, args.overpass_api)
+
+
+def add_args(parser):
+    parser.add_argument(
+        "bbox",
+        help="The target bounding box in format minlon,minlat,maxlon,maxlat. "
+             "Get the bounding box from https://boundingbox.klokantech.com/.",
+    )
+    parser.add_argument("-auth",
```

Comment:
Common notation for arguments is `"-a", "--auth"`: one-letter shortcut and the full version with two dashes. Please update this one (and readme).

Reply:
yes, sorry, I mixed that up. Wanted to ensure its noted as a flag but flags also have `--`. Resolved in https://github.com/Zverik/osm_to_sandbox/commit/ba12e74e963b04d9aa18093cbd5e765cae4e7ac1